### PR TITLE
fix(agw): wrong state transition in Sercomm

### PR DESCRIPTION
**Pull request by WojSad**
Signed-off-by: Wojciech Sadowy <wojciech.sadowy@freedomfi.com>

fix(agw): wrong state transition in Sercomm

## Summary

After acs-triggered reboot Sercomm's (freedomfi_one) state machine
was trying to enter an undefined state 'wait_empty' raising an error
Changed to 'wait_inform' which is the type of message that is actually sent after
the reboot

## Test Plan

tests updated

### CODE REVIEW

**Comment by xbend** [comment]
@ulaskozat 
This change should be reviewable now. WPS python checks will not pass on AGW python code, mme test failure is related to a bug present in master cpp code.

File: lte/gateway/python/magma/enodebd/devices/freedomfi_one.py [hunk 1]
```diff
@@ -946,7 +946,7 @@ def _set_default_params(self, desired_cfg):
             **SASParameters.defaults,
         }
         for name, val in defaults.items():
-            desired_cfg.set_parameter(name, val)
+            desired_cfg.set_parameter(name, value=val)
```
Comment (ardzoht) [inline lte/gateway/python/magma/enodebd/devices/freedomfi_one.py]: pass keyword argument to `name` as well to be consistent

**Review by ardzoht — CHANGES_REQUESTED** [review]
minor change requested, once that's updated I'll approve the PR

File: lte/gateway/python/magma/enodebd/devices/freedomfi_one.py (hunk 1 above)
Comment (WojSad) [reply lte/gateway/python/magma/enodebd/devices/freedomfi_one.py]: done

**Comment by xbend** [comment]
@ardzoht 
Requested changes have been addressed.

**Review by ardzoht — APPROVED** [review]
lgtm

**Pull request merged**